config/Dockerfiles/JenkinsfileRelease

```diff
@@ -0,0 +1,182 @@
+/**
+ * CD LinchPin Release
+ *
+ * This job pulls down the git repository when a new version has been
+ * provided, tags the release version and pushes a new release to pypi.org
+ */
+
+import com.cloudbees.plugins.credentials.Credentials
+
+// Execution ID for this run of the pipeline
+executionID = UUID.randomUUID().toString()
+// Use repoName everywhere for dirs and repo values
+repoName = 'jenkins-test'
+
+
+env.ghprbGhRepository = env.ghprbGhRepository ?: 'herlo/' + repoName
+env.ghprbActualCommit = env.ghprbActualCommit ?: 'develop'
+env.ghprbPullAuthorLogin = env.ghprbPullAuthorLogin ?: ''
+env.ghprbPullId = env.ghprbPullId ?: ''
+
+env.TARGET_BRANCH = env.TARGET_BRANCH ?: 'develop'
+
+OPENSHIFT_NAMESPACE = env.OPENSHIFT_NAMESPACE ?: 'continuous-infra'
+OPENSHIFT_SERVICE_ACCOUNT = env.OPENSHIFT_SERVICE_ACCOUNT ?: 'jenkins'
+
+
+// Pod name to use
+podName = repoName + executionID
+
+library identifier: "ci-pipeline@master",
+        retriever: modernSCM([$class: 'GitSCMSource',
+                              remote: "https://github.com/CentOS-Paas-SIG/ci-pipeline"])
+
+library identifier: "cico-pipeline-library@master",
+        retriever: modernSCM([$class: 'GitSCMSource',
+                              remote: "https://github.com/CentOS/cico-pipeline-library"])
+
+
+properties([
+  buildDiscarder(logRotator(artifactNumToKeepStr: '20', numToKeepStr: '20')),
+  [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/herlo/' + repoName],
+  parameters(
+    [
+      string(defaultValue: 'develop',
+             description: '',
+             name: 'ghprbActualCommit'),
+      string(defaultValue: '',
+             description: '',
+             name: 'sha1'),
+      string(defaultValue: '',
+             description: 'Pull Request Number',
+             name: 'ghprbPullId'),
+      string(defaultValue: '',
+             description: 'Pull Request Author username',
+             name: 'ghprbPullAuthorLogin'),
+      string(defaultValue: '',
+             description: 'Git Hub Repository',
+             name: 'ghprbGhRepository'),
+      string(defaultValue: 'stable',
```

Comment:
you can take this variable out since you don't use it below.  Or update the containerTemplate below to use it.

Reply:
@p3ck I'm not sure how I might use this in the containerTemplate, assuming you are meaning the ghprbGhRepository variable. But removing it would mean that I can't use it below, correct?

Reply:
I'm talking about the stable tag.